client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -243,8 +256,8 @@ class ActivityLogTasklist extends Component {
 
 		each( itemsWithUpdate, item => {
 			const { slug, updateStatus, type, name } = item;
-			// Finds in either prevProps.pluginsWithUpdate or prevProps.themesWithUpdate
-			const prevItemWithUpdate = find( prevProps[ `${ type }sWithUpdate` ], { slug } );
+			// Finds in either prevProps.pluginWithUpdate or prevProps.themeWithUpdate
```

Comment:
should we also add the coreWithUpdate here?

Reply:
Yess, will update.